sound/soc/codecs/rt715-sdw.c

```diff
@@ -198,8 +198,6 @@ static int rt715_read_prop(struct sdw_slave *slave)
 	unsigned long addr;
 	struct sdw_dpn_prop *dpn;
```

Comment:
Hi @jack-cy-yu,

Please explain why this commit is needed. Also, please format the commit subject adding some 'tags' to specify the subsystem that is modified by your patch.

Have a look at previous commits git log  sound/soc/codecs/rt715-sdw.c

Reply:
Hi @dbaluta ,

I got a message from @plbossart  as below
"Forgot to mention that I see the following warning for rt715, likely some cleanup necessary as well.
[ 5120.662052] rt715 sdw:3:25d:715:0:1: DP0 node not found!!"

I reviewed the rt715 driver and found this function could be the reason for the warning message, so I removed it from current code just as rt711/rt1308 did.